### DISCUSSION TITLE
Add "WebRTC Signal Server" into the Realtime option as it will open up for any user to add Audio/Video as well.... !!!?

### DISCUSSION THREAD

**Discussion by rahul3v** [supabase/supabase#11427] [Feature Requests]
Adding "WebRTC Signal Server" will open up for any user to add Audio/Video features into their Application easily. 

You can add/use https://peerjs.com/peerserver
which is open-source (https://github.com/peers/peerjs-server)

Hope to get it sooner :)

**Comment by w3b6x9** [comment]
@rahul3v good suggestion! The Supabase Realtime team is actively exploring this. Out of curiosity, what prompted you to suggest this? Are you trying to build an app that requires WebRTC? If so, what's the app you're building or want to build?

**Reply by rahul3v** [reply]
Hello @w3b6x9 , 
Well I am building something that required WebRTC server to add Video/Audio feature of my own and to avoid managing my own WebRTC Server :)

And It's also very useful to make direct communication to the other users without much hitting server like using web-sockets all the time
